### PR TITLE
Ch fix pep8 errors

### PR DESCRIPTION
**Pull request by dennomwas**
**What does the PR do** 

Make changes to comply with Pep8

**Description of tasks completed** 
Fix code on the following directories to comply with Pep8:
*wger_vulcan/wger/nutrition
*wger_vulcan/wger/software

**How should this be manually tested?**
Open terminal, cd to the program directory and run the following commands
```diff
pip install pep8
```
```diff
pep8 <root folder path>
```

**Pivotal tracker stories** 
#142465229

### CODE REVIEW

File: wger/core/migrations/0010_auto_20170330_1116.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-30 08:16
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
```
Comment (layodi) [inline wger/core/migrations/0010_auto_20170330_1116.py:7]: Avoid making changes to migrations.

**Review by layodi — COMMENTED** [review]
The PR format is missing some of the aspects that should be included.
https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices

File: wger/core/migrations/0010_auto_20170330_1116.py (hunk 1 above)
Comment (dennomwas) [reply wger/core/migrations/0010_auto_20170330_1116.py:7]: Noted...

Comment (dennomwas) [reply wger/core/migrations/0010_auto_20170330_1116.py:7]: @andela-layodi I have updated the PR description, let me know if it is in order....

Comment (layodi) [reply wger/core/migrations/0010_auto_20170330_1116.py:7]: Yes it is, could you try putting the commands like `pip install pep8` using markdown, it makes them more readable and like distinct.

**Review by layodi — APPROVED** [review]
Looks good.

**Pull request merged**